worksheet_B/README.md

```diff
@@ -0,0 +1,60 @@
+INPUT random 5 letter word from given list i.e."Sheep", "Towel", "Light", "Learn", "Third", "Every"
+
+DECLARE word variable
+
+DECLARE correctanswer variable
+
+DECLARE guesscount variable
+
+SET correctanswer to False
+
+SET guesscount to 0
+
+WHILE guesscount is less than 4
+
+    DISPLAY 'You have a 5 letter word'
+
+    DISPLAY 'Can you guess what it is? Enter here:'
+
+	  User INPUT guess
+
+	  DECLARE guess variable
+
+	  IF guess is equal to 5
+		    ARRAY word variable, DECLARE as 'wordlist'
+		    ARRAY guess variable, DECLARE as 'guesslist'
+		    SET matching to 0
+
+        FOR i in range(0, 5):
+		  IF guesslist [i] is the same as wordlist [i]:
+			SET matching to matching + 1
+		IF matching is 0
```

Comment:
Indentation is unclear here as you have mixed tabs and spaces -- the GitHub website displays a tab as 8 spaces, but I'm guessing your text editor uses a different number of spaces